SilKit/source/config/ParticipantConfiguration.schema.json

```diff
@@ -768,6 +768,26 @@
         }
       },
       "additionalProperties": false
+    },
+    "Experimental": {
+      "type": "object",
+      "description": "Experimental configuration",
+      "properties": {
+        "TimeSynchronization": {
+          "type": "object",
+          "description": "Configuration related to time synchronization",
+          "properties": {
+            "AnimationFactor": {
+              "type": "number",
+              "description": "Wall clock slow down factor",
```

Comment:
```suggestion
              "description": "The animation factor describes how fast the simulation is allowed to run, relative to the local wall clock. For example, if the value is 0.1 SIL Kit will try to run the simulation 10 times faster than the wall clock. The default value 0.0 runs the simulation as fast as possible, i.e., the current behavior.",
```

Reply:
This is just a proposal. I'm not sure if the `Wall clock slow down factor` is worse than this.